.github/workflows/i18n_check.yml

```diff
@@ -44,11 +49,15 @@ jobs:
 
             # Does not need branch matching as only analyses this layer
             - name: Install Deps
-              run: "yarn install --frozen-lockfile"
+              env:
+                  YARNCWD: ${{github.event.inputs.path}}
```

Comment:
You could just use `working-directory: ${{ inputs.path }}` rather than env+--cwd